addons/account/tests/test_taxes_tax_totals_summary.py

```diff
@@ -478,12 +457,8 @@ def test_taxes_l10n_br(self):
                         },
                     ],
                 }
-                self.assert_tax_totals_summary(document, expected_values)
-                invoice = self.convert_document_to_invoice(document)
-                self.assert_invoice_tax_totals_summary(invoice, expected_values)
-
+                yield 1, self.populate_document(document_params), expected_values
```

Comment:
What is the reason to have a generator like this compared to having a helper that takes these as arguments? You could run the subtest inside that helper too.

I find this to be less readable without any reason :shrug: